package.json

```diff
@@ -2,27 +2,21 @@
   "name": "pcb-stackup",
   "version": "0.1.0",
   "description": "Creates PCB stackup (top and bottom view) SVG renders from gerber-to-svg converters",
-  "main": "lib/index.js",
-  "directories": {
-    "test": "test"
-  },
+  "main": "index.js",
   "scripts": {
-    "lint": "eslint lib/*.js test/*.js",
-    "test": "istanbul cover -x 'integration/*' -x 'example/*' --include-all-sources _mocha",
+    "lint": "eslint *.js integration/*.js example/*.js",
+    "test": "istanbul cover _mocha",
```

Comment:
For future reference, to exclude a file (like test.js) from coverage, you can do:

```
istanbul cover -x test.js node_modules/.bin/_mocha
```